geode-core/src/main/java/org/apache/geode/internal/cache/InitialImageOperation.java

```diff
@@ -1100,6 +1105,12 @@ protected Set<Object> processReceivedRVV(RegionVersionVector remoteRVV,
         }
       }
     }
+    if (foundIds.size() > 0) {
```

Comment:
`foundIds` always has at least one element. Because of line 1067 `foundIds.add(region.getVersionMember());`.  So this `if ` check is not necessary.

Reply:
fixed.